### PR TITLE
Fix for broken Try in PWD button

### LINKED ISSUE #91: Fix Try in PWD button
`Try in PWD` (Play with Docker) button is broken since Blaze v2.0.0 update. It needs to be fixed by adding a separate configuration file.
A good repository for some reference is https://github.com/vegasbrianc/prometheus
Also, check out the `docker-compose` file to learn more about how the project is structured in terms of Docker images.

### PR DESCRIPTION
**Pull request by minaorangina**
Adds a separate config file to prevent docker swarm error (`docker-stack.yml`). Closes #91

### CODE REVIEW

File: docker-stack.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  blaze-server:
+    image: node:10.16.3
```
Comment (blenderskool) [inline docker-stack.yml:4]: Why are we using the standard node image here? Blaze has its own server and client images.

**Review by blenderskool — CHANGES_REQUESTED** [review]
@minaorangina There seems to be a problem here, tested it by visiting this link https://labs.play-with-docker.com/?stack=https://raw.githubusercontent.com/minaorangina/blaze/master/docker-stack.yml and doesn't seem to work as expected

**Comment by minaorangina** [comment]
@blenderskool What do you observe when you click the link? An error message?

Also, do you know roughly when the button stopped working? You mentioned in the issue it was after the 2.0 release, but I haven't found a commit so far where it's working. I'd like something to compare with.

File: docker-stack.yml (hunk 1 above)
Comment (minaorangina) [reply docker-stack.yml:4]: 👍 I'll change it

**Pull request closed without merging**